sycl/source/detail/windows_os_utils.hpp

```diff
@@ -0,0 +1,28 @@
+//==-- windows_os_utils.hpp - Header file with common utils for Windows --==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===--------------------------------------------------------------------===//
+
+#pragma once
+
+#include <shlwapi.h>
+
+using OSModuleHandle = intptr_t;
+static constexpr OSModuleHandle ExeModuleHandle = -1;
```

Comment:
```suggestion
constexpr OSModuleHandle ExeModuleHandle = -1;
```

or `inline conxtexpr` if necessary (I think it's `inline` by default).

Reply:
Fixed.